staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -173,21 +174,41 @@ func NewDefaultPluginHandler(validPrefixes []string) *DefaultPluginHandler {
 func (h *DefaultPluginHandler) Lookup(filename string) (string, bool) {
 	for _, prefix := range h.ValidPrefixes {
 		path, err := exec.LookPath(fmt.Sprintf("%s-%s", prefix, filename))
-		if err != nil || len(path) == 0 {
-			continue
+		if err != nil {
+			if strings.Contains(err.Error(), "current directory") {
+				err = nil
+			} else if len(path) == 0 {
+				continue
+			}
 		}
 		return path, true
 	}
-
 	return "", false
 }
 
+func Command(name string, arg ...string) *exec.Cmd {
+	cmd := &exec.Cmd{
+		Path: name,
+		Args: append([]string{name}, arg...),
+	}
+	if filepath.Base(name) == name {
+		lp, _ := exec.LookPath(name)
+		if lp != "" {
```

Comment:
could use the same error checker helper here just to make sure we're not getting some other error we actually want to surface:

```go
lp, err := exec.LookPath(name)
if lp != "" && !shouldSkipOnLookPathErr(err) {
  cmd.Path = lp
```